src/Streaming/Internal/Process.hs

```diff
@@ -94,7 +94,7 @@ import Data.Functor.Sum
 import qualified Data.IntSet as IntSet
 import qualified Data.Sequence as Seq
 import qualified Data.Set as Set
-import Data.Unrestricted.Linear
+import Data.Unrestricted.Linear (Consumable, Ur (..), lseq)
```

Comment:
This is a little suspicious. What happened?

Reply:
When I started this PR, `Replicator`, and its `next/next#` methods were defined in `Data.Unrestricted.Linear.Internal.Replicator`, so they were publicly reexported by `Data.Unrestricted.Linear`.
`src/Streaming/Internal/Process.hs` also defines a `next` function, creating a name clash.

Reply:
So you can revert this, now? Please do.